packages/one-app-runner/__tests__/bin/__snapshots__/one-app-runner.spec.js.snap

```diff
@@ -30,12 +29,6 @@ Array [
   ----container-name        Assign a container name with the --name option
                                                                         [string]
   --help                    Show help                                  [boolean]",
-  ],
-  Array [],
-  Array [
-    "Missing dependent arguments:
- dev-endpoints -> modules",
```

Comment:
To solve the issue with yargs on GH actions, how are we testing for this error instead of the entire snapshot?

Reply:
You brought up a good point, I added a check for the unchanging portion of the error message that describes the implicit command. It checks `dev-endpoints -> modules` and likewise for parrot middleware.

https://github.com/americanexpress/one-app-cli/pull/198/files#diff-b1738cb759b985fc37b65c0bbbade00b4397cd0af25b18d0d823c1c032862b8aR265-R277